Mail2Bug/Email/EWS/EWSIncomingAttachmentConverter.cs

```diff
@@ -0,0 +1,134 @@
+﻿﻿using log4net;
+using Microsoft.Exchange.WebServices.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Mail2Bug.Email.EWS
+{
+    public class EWSIncomingAttachmentConverter
+    {
+        // TFS drops large base64 inline images: http://johannblais.blogspot.com/2015/08/migrate-tfs-repro-steps-field-to.html
+        private const int MaxBase64Length = 22 * 1024; // This a safe guess
+
+        private readonly EmailMessage _message;
+
+        private string _bodyText;
+        private List<IIncomingEmailAttachment> _attachments = null;
+
+        public EWSIncomingAttachmentConverter(EmailMessage message)
+        {
+            _message = message;
+        }
+
+        public void ProcessAttachments(bool convertInlineAttachments = false)
+        {
+            _bodyText = _message.Body.Text ?? String.Empty;
+            _attachments = new List<IIncomingEmailAttachment>();
+
+            bool doInlining = !String.IsNullOrEmpty(_bodyText) && convertInlineAttachments;
+
+            foreach (var ma in _message.Attachments)
+            {
+                if (doInlining)
```

Comment:
For a single-line-body if clause (without an else), I think it's fine to keep it without braces and just hoist the body to the same line as the 'if' - that makes it less likely to hit the issue mentioned above